NTupleAnalyzerHH_nanoAOD.py

```diff
@@ -800,21 +850,24 @@ def JetsFromLQ(T):
 	for n in range(T.nJet):
 		if  ((T.Jet_jetId[n] & 0x2) > 0) and T.Jet_pt[n]>30 and abs(T.Jet_eta[n])<2.4 : #morse only use jets that pass tightid, fixme check jet eta
 			m = TLorentzVector()
-			m.SetPtEtaPhiM(T.Jet_pt[n],T.Jet_eta[n],T.Jet_phi[n],0)
+			m.SetPtEtaPhiM(T.Jet_pt[n],T.Jet_eta[n],T.Jet_phi[n],T.Jet_mass[n])
```

Comment:
This looks like a problem in the indentation compared to the old version, please check.